helm/reporting-agent-chart/templates/agent/1-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "agent.fullname" . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "agent.fullname" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- kind: ServiceAccount
+  name: {{ template "agent.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  
+
+--- 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "agent.fullname" . }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - daemonsets
+  - deployments
+  - ingresses
+  - namespaces
+  - pods
+  - replicasets
+  - services
+  - statefulsets
```

Comment:
I would add only `pods` for now. We are not scraping anything else for now

Reply:
but if we add more resources type in the configmap, we need to change the RBAC, 

let me try to do it in parallel. Like the same resources in values are use in script and in RBAC, WDYT?

Reply:
yeah we will need. But now we are not using it. you can reuse `Values.types` here to we dont have two places. I dont see this is going to change soon

Reply:
I've set it it like this already, check the latest code. it's working